### ISSUE TITLE
Forge.exe complains it needs JRE 11.0.1 even though I have it installed.

### ISSUE THREAD

**Issue by BigCasper** [Card-Forge/forge#6290]
Forge doesn't launch correctly and opens a window saying "This Application needs Java Runtime Enviroment 11.0.1" and opens the oracle site to download the JDK 21 (for some reason).

I have the JRE 11.0.24 installed (which should be backwards compatible with 11.0.1).

Reproduction:
Have the 11.0.24 JRE installed.
Run Forge.exe

**Comment by kevlahnota** [comment]
> Forge doesn't launch correctly and opens a window saying "This Application needs Java Runtime Enviroment 11.0.1" and opens the oracle site to download the JDK 21 (for some reason).
> 
> I have the JRE 11.0.24 installed (which should be backwards compatible with 11.0.1).
> 
> Reproduction: Have the 11.0.24 JRE installed. Run Forge.exe

seems your default Java is set to Java 8 (1.8), try this:
1. Uninstall all Java Versions
2. Install Java 17 (or higher)
3. Run Forge.exe (Desktop UI) / Forge-adventure.exe (Mobile UI)

**Comment by BigCasper** [comment]
> > Forge doesn't launch correctly and opens a window saying "This Application needs Java Runtime Enviroment 11.0.1" and opens the oracle site to download the JDK 21 (for some reason).
> > I have the JRE 11.0.24 installed (which should be backwards compatible with 11.0.1).
> > Reproduction: Have the 11.0.24 JRE installed. Run Forge.exe
> 
> seems your default Java is set to Java 8 (1.8), try this:
> 
> 1. Uninstall all Java Versions
> 2. Install Java 17 (or higher)
> 3. Run Forge.exe (Desktop UI) / Forge-adventure.exe (Mobile UI)

I don't even have Java 8 installed. I got 11, 17 and 21.

**Comment by kevlahnota** [comment]
> > > Forge doesn't launch correctly and opens a window saying "This Application needs Java Runtime Enviroment 11.0.1" and opens the oracle site to download the JDK 21 (for some reason).
> > > I have the JRE 11.0.24 installed (which should be backwards compatible with 11.0.1).
> > > Reproduction: Have the 11.0.24 JRE installed. Run Forge.exe
> > 
> > 
> > seems your default Java is set to Java 8 (1.8), try this:
> > 
> > 1. Uninstall all Java Versions
> > 2. Install Java 17 (or higher)
> > 3. Run Forge.exe (Desktop UI) / Forge-adventure.exe (Mobile UI)
> 
> I don't even have Java 8 installed. I got 11, 17 and 21.

Did you try launching the forge.cmd instead of forge exe?

**Comment by BigCasper** [comment]
yeah. it opened cmd for a sec and then closed it. that's it.

**Comment by matenwego** [comment]
I'm having this same issue, I have `openjdk 23.0.1 2024-10-15` in Windows. It's the only installation I have installed. Running `forge.cmd` works with this output in cmd:
```
Forge v.1.6.65 (forge.GuiDesktop@58ea606c)
Windows 11 (10.0 amd64)
Oracle Corporation (Java 23.0.1+11-39, OpenJDK 64-Bit Server VM)
```

**Comment by kevlahnota** [comment]
> I'm having this same issue, I have `openjdk 23.0.1 2024-10-15` in Windows. It's the only installation I have installed. Running `forge.cmd` works with this output in cmd:
> 
> ```
> Forge v.1.6.65 (forge.GuiDesktop@58ea606c)
> Windows 11 (10.0 amd64)
> Oracle Corporation (Java 23.0.1+11-39, OpenJDK 64-Bit Server VM)
> ```

This is fixed on 2.0.00 snapshot since java 16 and below support is dropped. Java 17 is the minimum on 2.0.00
